.github/workflows/scrape.yml

```diff
@@ -30,7 +30,9 @@ jobs:
         pip3 install -r requirements.txt
     - name: Generate battleground-state-changes.txt
       run: |-
-        ./print-battleground-state-changes > tmp
+        echo "Data retrieved at approximately $(date)" > tmp
+        echo "" >> tmp
+        ./print-battleground-state-changes >> tmp
```

Comment:
Can you have the script print this? Seems much simpler.

Reply:
Sure, one moment. I'll have it put UTC time (the rest of the script) and EST.

Reply:
Nevermind on EST, timezones suck and doing this right in python was getting uglier than I wanted without adding a dependency